library.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "compact25519",
+    "version": "1.0.0",
```

Comment:
will this pickup a tag in the repo? (and the last version is 1.1.0)

Reply:
It doesn't. I fixed the version number to 1.1.0, but I think platformio is a bit annoying in that it expects this to be manually updated.